parts/k8s/kubernetescustomscript.sh

```diff
@@ -113,6 +113,7 @@ fi
 
 if [[ "${GPU_NODE}" = true ]]; then
     installGPUDrivers
+    ensureGPUDrivers
```

Comment:
Do we need another "gpu driver installed succesfully check" before we invoke ensureGPUDrivers?

Reply:
@tariq1890 not sure what you mean, installGPUDrivers is called right before and that should check that the driver is installed and install it if not.

Reply:
Is there a possibility of installGPUDrivers failing?

Reply:
yes in which case the script will exit non-zero at ensureGPUDrivers. What this PR is doing is separating the install part of the GPU drivers install from what needs to be done at runtime so that the VHD builder can call the install part only. The function itself is unchanged from what was previously there, it was just split into two sub functions which are called one after the other in CSE to preserve behavior.

However, looking through it it does look like installGPUDrivers is missing some exit codes so I look into adding those.